toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/TableTopSceneView.kt

```diff
@@ -102,26 +122,69 @@ fun TableTopSceneView(
     onDrawStatusChanged: ((DrawStatus) -> Unit)? = null,
     content: (@Composable TableTopSceneViewScope.() -> Unit)? = null
 ) {
+    var initializationStatus: TableTopSceneViewStatus by remember {
+        mutableStateOf(
+            TableTopSceneViewStatus.Initializing
+        )
+    }
```

Comment:
As mentioned above, could use helper function:
```suggestion
    var initializationStatus = rememberTableTopSceneViewStatus()
```